client-cli/src/command/wallet_command.rs

```diff
@@ -31,6 +37,43 @@ pub enum WalletCommand {
         )]
         wallet_type: WalletKind,
     },
+    #[structopt(name = "export", about = "Backup wallet to a file")]
+    Export {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: Option<String>,
+        #[structopt(
+            name = "from_file",
+            short = "f",
+            long = "from_file",
+            parse(from_os_str),
+            help = r#"json file contains a list of {"name": wallet_name, "auth_token": auth_token}"#
+        )]
+        from_file: Option<PathBuf>,
+        #[structopt(
+            name = "to_file",
+            short = "t",
+            long = "to_file",
```

Comment:
just 'to'?

Reply:
https://github.com/crypto-com/chain/pull/1106#discussion_r382356917